### PR TITLE
Add shared styles

### PR DESCRIPTION
**Pull request by chuckh**
Add the new Polymer 1.1.0 share styles approach. Addresses #267.

### CODE REVIEW

File: app/elements/my-greeting/my-greeting.html [hunk 1]
```diff
@@ -8,22 +8,14 @@
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 
+<link rel="import" href="../../styles/shared-styles.html">
```
Comment (moderndeveloperllc) [inline app/elements/my-greeting/my-greeting.html]: Is this line needed as it's already being imported in the elements.html?

Comment (chuckh) [inline app/elements/my-greeting/my-greeting.html]: Good catch. No it is not needed. I wil remove.

**Comment by addyosmani** [comment]
Thanks for throwing this together, @chuckh. I've been reviewing https://github.com/Polymer/polymer/issues/1876 https://github.com/Polymer/polymer/issues/2229#issuecomment-134603556 &  https://github.com/Polymer/polymer/pull/2198 to catch up on the new approach. 

Your changes LGTM here, however testing locally with 1.1.1 I'm seeing some differences between Chrome and Safari (Mac):



I've verified that master otherwise appears to be fine:



Could you do another pass on testing the change out to see what might be causing the discrepancy?

**Comment by chuckh** [comment]
@addyosmani did some more testing this morning. It worked for my on Chrome and Safari, but got the same results as you did but with Firefox instead.  I was using 1.1.1, but then I decide to try doing a bower update and that fixed it. Try doing bower update then try testing again.


### Results after bower update:

Safari (8.0.8 mac):
image

Firefox (40.0 mac): 
image

Chrome (44.0.2403 mac)
image

**Comment by addyosmani** [comment]
LGTM

**Pull request merged**